link/perf_event.go

```diff
@@ -147,48 +114,170 @@ func (pe *perfEvent) Close() error {
 	return nil
 }
 
+// perfEventLink represents a bpf perf link.
+type perfEventLink struct {
+	RawLink
+	pe *perfEvent
+}
+
+func (pl *perfEventLink) isLink() {}
+
+// Pinning requires the underlying perf event FD to stay open.
```

Comment:
Can you cut a ticket for this as well?

Reply:
https://github.com/cilium/ebpf/issues/597